runelite-client/src/main/java/net/runelite/client/plugins/microbot/shortestpath/Transport.java

```diff
@@ -326,4 +334,58 @@ private static boolean completedQuests(Transport transport) {
         }
         return true;
     }
+
+    public boolean handleSpiritTree() {
+        // Get Transport Information
+        String displayInfo = this.getDisplayInfo();
+        String objectName = this.getObjectName();
+        int objectId = this.getObjectId();
+        String action = this.getAction();
+        WorldPoint origin = this.getOrigin();
+        WorldPoint destination = this.getDestination();
+
+        System.out.println("Display info: " + displayInfo);
+        System.out.println("Object Name: " + objectName);
+        System.out.println("Object ID: " + objectId);
+        System.out.println("Action: " + action);
+        System.out.println("Origin: " + origin);
+        System.out.println("Destination: " + destination);
+
+        // Check if the widget is already visible
+        if (!Rs2Widget.isHidden(12255232)) {
```

Comment:
Try to use early returns to avoid code indention

` if (Rs2Widget.isHidden(12255232)) return false;`